src/styles/variables.css

```diff
@@ -9,6 +9,7 @@
   --pink: rgb(254, 94, 124);
   --golden: rgba(255, 176, 0); /* Think pending states (golden yellow) */
   --dark: rgb(60, 68, 77); /* Used for colony name display (colony simplified) */
+  --dark-1: 60, 68, 77; /* @TODO: Remove after refactoring entire file. */
```

Comment:
You might have noticed we can't use `color-mod` in the cdapp due it being removed from the relevant post-css plugin we're using.

This does the same thing. You use it like:
``` css
background-color: rgba(var(--dark), 0.5)
 ```